ext/appmap/appmap.c

```diff
@@ -17,7 +18,13 @@ static VALUE
 singleton_method_owner_name(VALUE klass, VALUE method)
 {
   VALUE owner = rb_funcall(method, rb_intern("owner"), 0);
+
+#if RUBY_API_VERSION_CODE < 30200
```

Comment:
Whoa

Reply:
It was pretty grotty before. The change they made in 3.2 allows it to be slightly less so.